cloudbuild-pr.yaml

```diff
@@ -21,37 +21,12 @@ steps:
   allowFailure: true
 
 - name: 'gcr.io/cloud-builders/docker'
-  args: [ 'build', '-t', 'us-docker.pkg.dev/${PROJECT_ID}/cluster-provision-dev/cluster-provision-image:latest',
```

Comment:
This we remove all our PR time validations. Is this intended ?

Reply:
Yup! Two reasons:
- This tool no longer takes an environment variables list as input
- If I'm not mistaken, these tests do what the other ones do